src/core/constructs/output_manager.py

```diff
@@ -287,7 +285,41 @@ def _print_component_resource_differences(
                 self._console.print(
                     f"        [bold yellow]Update:[/bold yellow][bold blue] {resource_diff.new_resource.name} ({resource_diff.new_resource.ruuid})[/bold blue]"
                 )
-
+                for key in resource_diff.new_resource.dict().keys():
+                    if resource_diff.previous_resource.dict().get(
+                        key
+                    ) != resource_diff.new_resource.dict().get(key):
+                        if type(resource_diff.previous_resource.dict().get(key)) == frozenset:
+                            event = resource_diff.previous_resource.dict().get(key)
+                            event2 = resource_diff.new_resource.dict().get(key)
+                            dict_prev = dict()
+                            dict_new = dict()
+                            for item in event:
+                                for key2 in item:
+                                    dict_new[key2] = item.get(key2)
+                            for item in event2:
+                                for key2 in item:
+                                    dict_prev[key2] = item.get(key2)
+                            for key2 in dict_prev:
+                                if dict_prev[key2] != dict_new[key2]:
+                                    self._console.print(
+                                        f"           [bold black]Diff {key}:[/bold black]  {dict_prev[key2]} -> {dict_new[key2]} "
+                                    )
+                        elif type(resource_diff.previous_resource.dict().get(key)) != str:
+                            for item in resource_diff.previous_resource.dict().get(key):
```

Comment:
Can convert to normal dict with `dict(resource_diff.previous_resource.dict().get(key))`